contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -2215,4 +2217,56 @@ check_windows_logon_length(char* input)
 		return true;
 	else
 		return false;
+}
+
+static bool
+check_windows_user(const char* login)
```

Comment:
Actually we can not, because, whenever we are updating the user catalog then the login is already in UPN format so we can not take advantage of whether it contains '\\' or not. We can only validate whether it is windows user or not by looking up the corresponding login in the login catalog. We can not take advantage of original_user_name as well because it may contain '\\' or it may not contain that. Also, if we want to add any input argument in the add_to_bbf_authid_user_ext function, then we will have to update all the occurrences, which can be a bit tricky. Kindly suggest any other better alternatives if present.

Reply:
Done!